usage/bigquery.go

```diff
@@ -36,14 +36,32 @@ type UsageSummaryRow struct {
 	StorageUsageMins  float64 `bigquery:"storage_usage_mins"`
 }
 
+type TotalUsageSummaryRow struct {
+	DateTs                time.Time `bigquery:"date_ts" json:"dateTs"`
+	DateS                 int64     `bigquery:"date_s" json:"dateS"`
+	WeekTs                time.Time `bigquery:"week_ts" json:"weekTs"`
+	WeekS                 int64     `bigquery:"week_s" json:"weekS"`
+	VolumeEth             float64   `bigquery:"volume_eth" json:"volumeEth"`
+	VolumeUsd             float64   `bigquery:"volume_usd" json:"volumeUsd"`
+	FeeDerivedMinutes     float64   `bigquery:"fee_derived_minutes" json:"feeDerivedMinutes"`
+	ParticipationRate     float64   `bigquery:"participation_rate" json:"participationRate"`
+	Inflation             float64   `bigquery:"inflation" json:"inflation"`
+	ActiveTranscoderCount int64     `bigquery:"active_transcoder_count" json:"activeTranscoderCount"`
+	DelegatorsCount       int64     `bigquery:"delegators_count" json:"delegatorsCount"`
+	AveragePricePerPixel  float64   `bigquery:"average_price_per_pixel" json:"averagePricePerPixel"`
+	AveragePixelPerMinute float64   `bigquery:"average_pixel_per_minute" json:"averagePixelPerMinute"`
```

Comment:
Also make sure none of these fields are nullable on the query, or the bigquery parser might explode when they are null and the destination field is not nullable

Reply:
Checking with @ecmulli . Btw, for now some `float64` fields are NUMERIC on the database so this is not working, I could use `big.Rat` but it's better to change them to float there as well

Reply:
Confirmed that they are not nullable

Reply:
Yeah if you need to change the types from the Go literal ones, the JSON serialization wouldn't work. Then you'd need a separate struct to read from bigquery and another one to write as JSON, like I did for viewership. Not the end of the world, but I agree that if this is a new table and there's no reason to use `NUMERIC`, could be easier to just change that